build.sbt

```diff
@@ -370,7 +370,7 @@ lazy val refinedBase = circeCrossModule("refined", mima = previousCirceVersion)
   .settings(
     libraryDependencies ++= Seq(
       "eu.timepit" %%% "refined" % refinedVersion,
-      "eu.timepit" %%% "refined-scalacheck" % refinedVersion % Test
+      "eu.timepit" %%% "refined-scalacheck_1.13" % refinedVersion % Test
```

Comment:
This has been changed because `refined-scalacheck` depends on ScalaCheck 1.14 but circe is still on 1.13.